src/backend/TrafficCourts/Common/Configuration/TicketStorageType.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TrafficCourts.Common.Configuration;
+
+public enum TicketStorageType
+{
+    InMemory,
```

Comment:
We should switch the order. Having ObjectStorage first will make it default if not set. Makes configuring in openshift less complex.

Reply:
Good thought.  I just copied what was in Citizen Api from [here](https://github.com/bcgov/jag-traffic-courts-online/blob/main/src/backend/TrafficCourts/Citizen.Service/Configuration/TicketStorageType.cs#L6).

Done, swapped.